concepts/generators/introduction.md

```diff
@@ -1,2 +1,96 @@
-#TODO: Add introduction for this concept.
+# Instructions

```

Comment:
Did you mean for this to be the exercise introduction/instructions?  If not, I think this should be `#introduction`.

Reply:
Whoops, yep that

Reply:
```suggestion
# Introduction
```